deployment/deploy_to_preview.sh

```diff
@@ -1,6 +1,10 @@
 #!/bin/bash
 set -u
 
+profile=production
+if [[ $IN_MEMORY == true ]]; then
+  profile=productionInMemory
+fi
```

Comment:
Ich finde eine variable setzen, und dann durch was anderes überschreiben, nicht so schön.
Schöner finde ich in if/else jeweils den Zweig zu besetzen.

Reply:
ist angepasst

Reply:
ok